sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -71,6 +73,8 @@ def __init__(
                 with weights proportional to the `potential_fn`-value.
             init_strategy_num_candidates: Number of candidates to to find init
                 locations in `init_strategy=sir`.
+            num_workers: number of cpu core used to parallelize initialization of mcmc
```

Comment:
We should make clear that `num_workers` does not only set the number of cores used for init but also the default number of cores used for sampling